services/QuillLMS/client/app/bundles/Shared/components/reactTable/reactTable.tsx

```diff
@@ -3,6 +3,8 @@ import { useTable, useSortBy, usePagination, useFilters, useExpanded, useGroupBy
 
 import ReactTablePagination from './reactTablePagination'
 
+const DEFAULT_PAGE_SIZE = 100
```

Comment:
Maybe we should just do 1 here? Since it seems like this will never come into play unless there is no data at all.